doc/source/cluster/running-applications/job-submission/index.md

```diff
@@ -1,14 +1,14 @@
 (jobs-overview)=
 
-# Ray Jobs
+# Ray Job Submission
 
 Once you have deployed a Ray cluster (on [VMs](vm-cluster-quick-start) or [Kubernetes](kuberay-quickstart)), you are ready to run a Ray application!
 
-The recommended way to run a Ray application on a Ray Cluster is to use *Ray Jobs*.
-Ray Jobs allow you to submit locally developed applications to a remote Ray Cluster for execution.
+The recommended way to run a Ray application on a Ray Cluster is to use *Ray Job Submission*.
+Ray Job Submission allows you to submit locally developed applications to a remote Ray Cluster for execution.
 It simplifies the experience of packaging, deploying, and managing a Ray application.
 
-A Ray Job consists of:
+When using Ray Job Submission, a Ray Job consists of:
```

Comment:
This is awkward.  Open to suggestions

Reply:
Should we change the structure of the nav?

**Nav:**
Applications guide
- Ray job
   - Get started with ray jobs
   - Key concepts
   - Submit ray jobs to clusters
   - Run ray jobs remotely via ray client
   - Job API reference

For this job page, shall we just provide a more complete definition of ray jobs and add links to those sub-pages above?

Reply:
Actually, I'd like to suggest the alternative; What about combining these two into one section

```
Submit ray jobs to clusters
Run ray jobs remotely via ray client

-> Running ray jobs to the clusters
```

And within one page we have 3 sections, driver + job submission + client. And explain when each of them is useful

Reply:
I've updated this, what do you think of the current update?  The overview page includes two sections, one for Ray Jobs API (recommended) and another for running jobs interactively (driver, and client)